### PR TITLE
Parental leave: Duration percent

### PR DESCRIPTION
**Pull request by madebynoam**
## What

Add percentage UI to parental leave duration slider

## Why

Matches latest design

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by arnifreyrs — DISMISSED** [review]
nice!

File: libs/shared/form-fields/src/lib/FieldDescription/FieldDescription.tsx [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+export const FieldDescription: FC<Props> = ({ description }) => {
   const getMarkup = () => {
     return {
       __html: description,
     }
   }
   return (
     <Text marginTop={2} marginBottom={1}>
-      <span dangerouslySetInnerHTML={getMarkup()} />
+      {/* <span dangerouslySetInnerHTML={getMarkup()} /> */}
```
Comment (arnifreyrs) [inline libs/shared/form-fields/src/lib/FieldDescription/FieldDescription.tsx:16]: hmm, dont we need dangerouslySetInnerHTML?

Comment (madebynoam) [reply libs/shared/form-fields/src/lib/FieldDescription/FieldDescription.tsx:16]: Oops that was from debugging, good catch :) Will fix now.

Comment (madebynoam) [reply libs/shared/form-fields/src/lib/FieldDescription/FieldDescription.tsx]: Should be fixed now.

**Review by arnifreyrs — APPROVED** [review]

**Pull request merged**